diagon-alley/src/Context/WishlistContext/Context.js

```diff
@@ -0,0 +1,25 @@
+import { createContext, useContext, useReducer } from 'react';
+import { useEffect } from 'react/cjs/react.development';
```

Comment:
import it directly from 'react' like in line no.1. Otherwise this will create some weird bugs, many have faced this

Reply:
Thank you for pointing this out